eZ/Publish/Core/MVC/Symfony/FieldType/RichText/Renderer.php

```diff
@@ -123,7 +123,17 @@ public function renderContentEmbed($contentId, $viewType, array $parameters, $is
         $isDenied = false;
 
         try {
-            $this->checkContent($contentId);
+            $content = $this->checkContent($contentId);
+
+            if (!$content->contentInfo->mainLocationId) {
```

Comment:
Looks a little hackish, but if it works...
(I mean, shouldn't trash status be something ContentInfo explicitly knows about? If the definition of trashed is "has no location" then it obviously works. Could be more explicit, but +1 to not change api in bug fixes unless absolutely necessary.)

Reply:
TL;DR; there is no status exposed in API for content, todo here is expose it but rather name archive as trash status on api (unlike SPI).